fiscalsim_us/variables/gov/states/mn/tax/income/amt/mn_amt.py

```diff
@@ -1,28 +1,25 @@
-from fiscalsim_us.model_api import *
-
-
-class mn_amt(Variable):
-    value_type = float
-    entity = TaxUnit
-    label = "Minnesota alternative minimum tax (AMT)"
-    unit = USD
-    definition_period = YEAR
-    reference = (
-        "https://www.revenue.state.mn.us/sites/default/files/2023-02/m1mt_21.pdf"
-        "https://www.revenue.state.mn.us/sites/default/files/2023-01/m1mt_22.pdf"
-    )
-    defined_for = StateCode.MN
-
-    def formula(tax_unit, period, parameters):
-        taxinc = tax_unit("mn_amt_taxable_income", period)
-        # calculate gross AMT amount
-        filing_status = tax_unit("filing_status", period)
-        p = parameters(period).gov.states.mn.tax.income
-        excess_taxinc = max_(0, taxinc - p.amt.income_threshold[filing_status])
-        fractional_taxinc = excess_taxinc * p.amt.income_fraction
-        fractional_threshold = p.amt.fractional_income_threshold[filing_status]

```

Comment:
@mitchellpound. It looks like this `fractional_income_threshold` parameter, which remains in the `parameters` directory, is no longer used in any variable in this PR. However, it looks like it has information that is not in any of the four `.yaml` files you added to this PR. We should check if we need this. In particular, the [`fractional_income_threshold.yaml`](https://github.com/TheCGO/fiscalsim-us/blob/main/fiscalsim_us/parameters/gov/states/mn/tax/income/amt/fractional_income_threshold.yaml) parameters file has multiple tax years of values. This parameter looks most like your new `std_deduct_phase_out.yaml` parameter in this PR. But the parameter name is curious to me. Is it the income amount at which the AMT standard deduction begins, or is it the AMT standard deduction amount itself?

Reply:
I had the parameters switched. The parameter `standard_deduct.yaml` is the deduction amount under the amt and `std_deduct_phase_out.yaml` is the income threshold when the deduction starts phasing out. They should be fixed now.